.github/workflows/test-all.yml

```diff
@@ -206,7 +206,7 @@ jobs:
       - name: Restore Cached Artifacts
         uses: actions/cache/restore@v4.0.0
         with:
-          key: v2-hermes-osx-bin-${{ matrix.flavor }}-${{ needs.prepare_hermes_workspace.outputs.hermes-version }}-${{ needs.prepare_hermes_workspace.outputs.react-native-version }}
+          key: v2-hermes-artifacts-${{ matrix.flavor }}-${{ needs.prepare_hermes_workspace.outputs.hermes-version }}-${{ needs.prepare_hermes_workspace.outputs.react-native-version }}
```

Comment:
this is a fix to optimize the build. I was using the wrong prefix in the previous PR.